From dd66734ba7f24f41d72d5f465dd6fff02598087e Mon Sep 17 00:00:00 2001
From: Johannes 'josch' Schauer <josch@mister-muffin.de>
Date: Tue, 10 May 2016 07:36:38 +0200
Subject: [PATCH] lib/Sbuild/ResolverBase.pm: do not produce non-error output
 in architecture setup

sbuild-update setups up the resolver but not the logger. Hence, all
messages from the resolver will plainly go to stderr. This should not
happen as stderr for sbuild-update should be empty unless there really
is an error. We thus demote the messages to debug level. (closes: #823788)
---
 lib/Sbuild/ResolverBase.pm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/Sbuild/ResolverBase.pm b/lib/Sbuild/ResolverBase.pm
index b480cd2..86c48bf 100644
--- a/lib/Sbuild/ResolverBase.pm
+++ b/lib/Sbuild/ResolverBase.pm
@@ -183,7 +183,7 @@ sub get_foreign_architectures {
 
     if (!$foreignarchs)
     {
-        $self->log("There are no foreign architectures configured\n");
+        debug("There are no foreign architectures configured\n");
         return {};
     }
 
@@ -252,7 +252,7 @@ sub cleanup_foreign_architectures {
     my $session = $self->get('Session');
 
     if (defined ($session->get('Session Purged')) && $session->get('Session Purged') == 1) {
-	$self->log("Not removing foreign architectures: cloned chroot in use\n");
+	debug("Not removing foreign architectures: cloned chroot in use\n");
 	return;
     }
 
-- 
2.5.1

